src/css/_BRTemplate.scss

```
@@ -0,0 +1,7 @@
+#item-navigator-modal {
+  display: none
+}
+
+item-navigator-modal[mode=open] {
+  display: block;
+}
```

Add comment about why these are necessary.

```suggestion
#item-navigator-modal, item-navigator-modal[mode=open] {
  display: block;
}
```